MapboxNavigation/Resources/sv.lproj/Localizable.strings

```diff
@@ -44,7 +44,7 @@
 "RESUME" = "Återuppta";
 
 /* The text of a banner that appears during turn-by-turn navigation when route simulation is enabled. */
-"USER_IN_SIMULATION_MODE" = "Simulera navigation";
+"USER_IN_SIMULATION_MODE" = "Simulerar Navigation  %d×";
```

Comment:
I'm nowhere near proficient in Swedish, but it kind of looks like a preposition is missing here?

Reply:
Good catch